src/analysis/AnalysisNotificationManager.ts

```diff
@@ -1,6 +1,7 @@
 // This file is where any notifications not tied to a specific action are managed for leonardo-related resource
 // For example, if you load a page and your apps/runtimes are in an error state, this component is responsible for that notification
 
+import { append } from '@terra-ui-packages/core-utils';
```

Comment:
Went ahead and updated this import path since packaging Environments dependencies was a motivation.